requirements/requirements.in

```diff
@@ -1,3 +1,4 @@
+model_bakery>=1.13.0
```

Comment:
@adamchainz tests won't pass yet since I am unsure how to regenerate the dependency lockfiles via `requirements/compile.py`. Niether running via python directly nor via `tox -c requirements/compile.py` worked for me. I'm not familiar with `tox` so any direction here would be great.